cli/src/main/java/org/infinispan/cli/connection/Connection.java

```diff
@@ -73,6 +73,8 @@
 
    Collection<String> getBackupNames(String container) throws IOException;
 
+   Collection<String> getConnectorNames(String container) throws IOException;
```

Comment:
why the container? we don't support multiple containers and the connector has no association with connectors/enpoints.

Reply:
ok